doc/contributing/releases.md

```diff
@@ -216,6 +218,42 @@ Then, push to the staging branch to keep it up-to-date.
 $ git push upstream v1.x-staging
 ```
 
+<details>
+<summary>Security release</summary>
+
+Security releases with private patches need to be prepared in the `nodejs-private`
+GitHub organisation.
+
+Add the `nodejs-private` remote:
+
+```console
+$ git remote add private git@github.com:nodejs-private/node-private.git
+```
+
+For security releases, we generally try to only include the security patches.
+As there may already be commits on the `vN.x-staging` branch, it is preferable
+to checkout the `vN.x` branch and build up the release directly in a proposal
+branch.
+
+```console
+$ git checkout vN.x
+$ git reset --hard upstream/vN.x
+```
+
+The list of patches to include should be listed in the "Next Security Release"
+issue in `nodejs-private`. Ask the security release steward if you're unsure.
+
+When landing PRs from `nodejs-private`, you will need to manually apply the
```

Comment:
Maybe include a `cherry-pick` + `ammend` example?

Reply:
I applied a52b460, let me know if that works